tests/console/procps.pm

```diff
@@ -48,7 +56,52 @@ sub run {
         qr/PID\sTTY\s+TIME\sCMD\s+1\s\?\s+\d+:\d+:\d+\s(systemd|init)/);
     validate_script_output("top -b -n 1",
 qr/top - \d+:\d+:\d+ up\s+((\d+:\d+)|(\d+ \w+)|(\d+ \w+,\s+\d+:\d+)),\s+\d+ \w+,\s+load average: \d+.\d+, \d+.\d+, \d+.\d+\s+Tasks:\s+\d+\s+total,\s+\d+\s+running,\s+\d+\s+sleeping.*top/s);
-}
+
+    # Extend vmstat utility. Verifying the vmstat with options.
+    validate_script_output("vmstat -t",
+        qr/(procs\s-+memory-+\s-+swap-+\s-+io-+\s-+system-+\s-+cpu-+\s-+timestamp-+).*(\s+|\d+)+/s);
+    validate_script_output("vmstat -a",
+       qr/(procs\s-+memory-+\s-+swap-+\s-+io-+\s-+system-+\s-+cpu-+).*(\s+.*\s+inact\s+active\s*.*).*(\s+|\d+)+/s);
+    validate_script_output("vmstat -f", qr/(\d+\s+forks)/s);
+
+    #Running vmstat under light load and dumping the output to a file. Calculate and collect the minimum idle memory from the output.
+    assert_script_run("vmstat 2 10 > /tmp/output_light.log");
+    assert_script_run("cat /tmp/output_light.log");
+    my ($mem_lite_load_ref, $cpu_lite_load_ref) = read_memory_cpu("/tmp/output_light.log");
+    my @mem_lite_load = @$mem_lite_load_ref;    
+    my @cpu_lite_load = @$cpu_lite_load_ref;    
+    my $min_mem_lightload =  min(@mem_lite_load);
+    my $min_cpu_lightload =  min(@cpu_lite_load);
+    
+    # Increasing the load in the system using package stress-ng. 
+    zypper_call('in stress-ng');
+    assert_script_run("rpm -q stress-ng");
+    my $total_cpu = script_output("cat /proc/cpuinfo | grep processor | wc -l");
+    my $total_mem  = script_output("free -h | awk '/Mem\:/ { print \$2 }'");
+    if ($total_cpu==1)  {$cpu = $total_cpu;}  else  {$cpu = floor($cpu * 0.75);}
+    $vm = floor( $total_mem * 0.75);  
+    my $stressng_pkg =  "nohup stress-ng --cpu $cpu --vm $vm --vm-bytes 512M --timeout 60s&";
+    my $bg_pid = background_script_run($stressng_pkg);
+    validate_script_output("echo $bg_pid", sub {m/\d+/});
+   
+    # Running vmstat during the high load and storing the result in file. calculate the average of idle memory
+    assert_script_run("vmstat 2 10 > /tmp/output_load.log");
+    assert_script_run("pkill -9 stress-ng"); #kill the running background process stress-ng.  
```

Comment:
something really minor, try to keep a consistency on where you put the comments :)